dev-embedded/openocd/openocd-9999.ebuild

```diff
@@ -29,16 +28,17 @@ RDEPEND="
 	acct-group/plugdev
 	>=dev-lang/jimtcl-0.76:0=
 	cmsis-dap? ( dev-libs/hidapi )
-	jlink? ( >=dev-embedded/libjaylink-0.2 )
+	jlink? ( >=dev-embedded/libjaylink-0.2.0 )
 	usb? (
 		virtual/libusb:0
 		virtual/libusb:1
 	)
 	ftdi? ( dev-embedded/libftdi:= )"
 
-DEPEND="${RDEPEND}
+DEPEND="${RDEPEND}"
+BDEPEND="
+	sys-apps/texinfo
 	virtual/pkgconfig"
-[[ ${PV} == "9999" ]] && DEPEND+=" >=sys-apps/texinfo-5" #549946
 
 src_prepare() {
 	default
```

Comment:
amend the conditional after this line to be same `[[ ${PV} == *9999 ]]`

Reply:
Done